app/models/team.rb

```diff
@@ -133,21 +135,22 @@ def inactive?
     !active?
   end
 
+  # @note (Mohammed Seedat 2019-04-03) Original Team Name workaround
+  #   Team table has no way of storing historical data. On deactivation, the
+  #   Team's name is changed - name therefore stores deletion information.
+  #   To retrieve the Team name without any of the  deactivation information
+  #   we require this string replacement workaround
   def original_team_name
-    team_name = name.remove('DEACTIVATED')
-    team_name.remove(deleted_at.to_s)
+    name.remove(DEACTIVATED_LABEL, /@\((.)*\)/).strip
```

Comment:
I think this regex needs a comment...?

Reply:
Added additional note to explain string format scheme